plugins/esi/fetcher/CMakeLists.txt

```diff
@@ -16,6 +16,6 @@
 #######################
 
 add_library(fetcher STATIC HttpDataFetcherImpl.cc)
-target_link_libraries(fetcher PUBLIC esi-common libswoc)
-target_include_directories(fetcher PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}")
+target_link_libraries(fetcher PUBLIC esi-common libswoc::libswoc)
+target_include_directories(fetcher PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}" "${libswoc_INCLUDE_DIRS}")
```

Comment:
We should not be both linking the target and explicitly dumping its `INCLUDE_DIRS` into `target_include_directories`, as linking the target should already do this.